lib/types/CredentialIssuance.types.ts

```diff
@@ -57,7 +55,26 @@ export interface JWK {
   n?: string;
 }
 
-export type Alg = 'ES256' | 'EdDSA';
+export interface ProofOfPossessionCallbackArgs {
+  kid: string;
+  header: JWTHeader;
+  payload: JWTPayload;
+}
+
+export interface ProofOfPossessionOpts {
+  proofOfPossessionCallback: JWTSignerCallback;
+  proofOfPossessionVerifierCallback?: JWTVerifyCallback;
+  proofOfPossessionCallbackArgs: ProofOfPossessionCallbackArgs;
+}
+
+export enum Alg {
+  ES256 = 'ES256',
```

Comment:
ES256K is missing. Also using a type for what endsup in a payload is a bit more natural. But as long as the enum names are exactly the same as the string value it is okay

Reply:
added `ES256K`